packages/legacy/core/App/screens/Splash.tsx

```diff
@@ -104,140 +36,9 @@ const Splash: React.FC = () => {
     },
   })
 
-  // navigation calls that occur before the screen is fully mounted will fail
-  // this useeffect prevents that race condition
-  useEffect(() => {
-    setMounted(true)
-  }, [])
-
-  useEffect(() => {
```

Comment:
With this change, what does the user see while state is being loaded for the first time on a cold start?

Reply:
I think I may need to add this back then. It's a bit confusing but we have a splash screen imported from a NPM package and our Splash screen. I assumed there was a bug in the NPM version that is meant to show the app logo screen until the "state" is loaded at which point the app can then continue. Maybe I need to update Splash to show in two different ways.